easybuild/easyconfigs/i/Infernal/Infernal-1.1.4-foss-2022b.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'ConfigureMake'
```

Comment:
@Ghepardo could you restore the attribution and license information from previous versions please?

Reply:
Of course - I've submitted an update to this PR.  (I must admit I didn't have a lot of faith in those initial comment blocks generally - other contributors seem to have copied them blindly for certain new easyconfigs.)

Reply:
Thanks.